main.tf

```
@@ -100,3 +100,27 @@ resource "aws_vpc_ipv6_cidr_block_association" "default" {
     }
   }
 }
+
+resource "aws_default_route_table" "default" {
+  count                  = local.enabled && var.adopt_default_route_table ? 1 : 0
+  default_route_table_id = aws_vpc.default[0].default_route_table_id
+  tags = merge(module.label.tags, {
+    Name = join(module.label.delimiter, [module.label.id, "default"])
+  })
```

Please put this in a local such as `default_route_table_tags` and use it here

```suggestion
  tags = local.default_route_table_tags
```